ide/views.py

```diff
@@ -89,32 +89,51 @@ def save_to_db(request):
         net_name = request.POST.get('net_name')
         user_id = request.POST.get('user_id')
         next_layer_id = request.POST.get('nextLayerId')
-        public_sharing = True
         user = None
+        public_sharing = True
```

Comment:
Why was this necessary?

Reply:
This was necessary because then the model would not be shared and hence the canvas will not render the comments and sharing stuff on the model.